training/src/main/java/wolox/training/repositories/BookRepository.java

```diff
@@ -0,0 +1,13 @@
+package wolox.training.repositories;
+
+import java.util.List;
+import org.springframework.data.repository.Repository;
+import org.springframework.stereotype.Component;
+import wolox.training.models.Book;
+
+@Component
+public interface BookRepository extends Repository<Book, Long> {
```

Comment:
Check out the [CrudRepository](https://docs.spring.io/spring-data/commons/docs/current/api/org/springframework/data/repository/CrudRepository.html) interface, whose "findBy" methods return Optional< Object > (in this case, Optional<Book>) rather than the Object itself. This allows you to avoid returning null values :)